src/main/scala/com/gu/identity/play/CookieBuilder.scala

```diff
@@ -0,0 +1,33 @@
+package com.gu.identity.play
+import org.joda.time.{DateTime, Duration}
+import play.api.libs.functional.syntax._
+import play.api.libs.json.Reads._
+import play.api.libs.json._
+import play.api.mvc.Cookie
+
+import scala.util.Try
+
+object CookieBuilder {
+  def fromGuestConversion(json: JsValue, domain: Option[String] = None): Option[Seq[Cookie]] = {
+    implicit val cookieRead: Reads[Cookie] = (
+      (JsPath \ "key").read[String] and
+        (JsPath \ "value").read[String]
+      )(Cookie.apply(_, _))
+
+    for {
+      expirationString <- (json \ "cookies" \ "expiresAt").asOpt[String]
+      expiration <- Try { new DateTime(expirationString) }.toOption
+      maxAge = new Duration(DateTime.now, expiration).getStandardSeconds.toInt
+      cookies <- (json \ "cookies" \ "values").asOpt[Seq[Cookie]]
+      guuCookie <- cookies.find(_.name == "GU_U")
+      scguuCookie <- cookies.find(_.name == "SC_GU_U")
+      scgulaCookie <- cookies.find(_.name == "SC_GU_LA")
```

Comment:
The code above is constrained to the 3 known cookies, and I'd prefer that we match the [Frontend approach](https://github.com/guardian/frontend/blob/b17f08a/identity/app/services/PlaySigninService.scala#L19-L24) which honours _whatever_ list of cookies the Identity API response wants us to drop (tho' I agree that ideally the information about `httpOnly` and `secure` would be included in the API response, to reduce the amount of magic done by respecting `SC_`).